controllers/dellcsivolumegroupsnapshot_controller.go

```diff
@@ -354,7 +427,7 @@ func (r *DellCsiVolumeGroupSnapshotReconciler) bindContentToSnapshot(
 	if err := r.Get(ctx, nameSpacedName, vs); err != nil {
 		log.Error(err, "VG Snapshotter vg created ok  get VolumeSnapshot error")
 		r.EventRecorder.Eventf(vs, common.EventTypeWarning, common.EventReasonUpdated, "Failed to get newly created VolumeSnapshot %s in vg %s. error : %s", volumeSnapshotName, vgName, err.Error())
-		failedMap[s.Name] = fmt.Sprintf("VG Snapshotter vg created ok unable to get newly created VolumeSnapshot %s in %s", volumeSnapshotName, vgName)
+		failedMap[volumeSnapshotName] = fmt.Sprintf("VG Snapshotter vg created ok unable to get newly created VolumeSnapshot %s in %s", volumeSnapshotName, vgName)
```

Comment:
I would rephrase the string. "VG created successfully, but unable to get ...."

Reply:
we deliberately prefix "VG Snapshotter vg created ok .." to help troubleshoot , grep

Reply:
I am OK with prefix, but just suggested a different way to make reading more legible since these are in the logs, etc.

Reply:
Dont take this the wrong way, I am not trying to add more work or criticize anything over here. My point was that, these are logs customers will see too. It does not hurt if we make them more legible. If users want to grep, they can grep for pretty much anything. These were my suggestions, you can feel free to ignore.